tests/test_polynomial.py

```
@@ -0,0 +1,988 @@
+
+class Test_Polynomial(unittest.TestCase):
+
+    def runTest(self):
+
+        np.random.seed(2599)
+
+        # Test basic construction
+        # Polynomial is a Vector subclass, so it should accept Vector-like inputs
+        # Coefficients are in decreasing order: [a, b, c] = a*x^2 + b*x + c
+        p1 = Polynomial([1., 2., 3.])  # x^2 + 2x + 3
+        self.assertEqual(p1.shape, ())
```

Ruff (0.14.7)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)